app/controllers/miq_task_controller.rb

```diff
@@ -334,6 +322,9 @@ def tasks_set_default_options
   # Create a condition from the passed in options
   def tasks_condition(opts, use_times = true)
     cond = [[]]
+
+    cond = add_to_condition(cond, "jobs.guid IS NULL", nil) unless vm_analysis_task?
```

Comment:
not for today (or possibly ever) but...
active record's `where()` already builds queries for us. seems odd we are reinventing this wheel